_pages/dev/tms.md

```diff
@@ -0,0 +1,205 @@
+# Tag Management System
+
+Spartacus introduced the Tag Management System (TMS) integration in version 3.2.
+Before you continue reading this page, it's highly recommended to first get yourself familiar with the [Events Service](./event-service), as TMS relies on it.
+
+The TMS integration allows you to specify which Spartacus' events should be passed to the configured TMS. Google Tag Manager (GTM) and Adobe Experience Platform (AEP) systems are supported out of the box, while the other tag managers can easily be plugged in.
+
+Spartacus supports running multiple TMS in parallel, and you can decide which events should be collected by each of the supported TMS systems.
+
+## Setup
+
+Each TMS system might require a different setup instruction, and it's not possible to cover every system here.
```

Comment:
> TMS system

That's a bit duplicate, as it reads "Tag Management System System". Perhaps we rephrase to: 
> Each Tag Management solution requires different...

Reply:
Good point, done.